### ISSUE TITLE
Screenshot capture text is included in the screenshot

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#289]
Originally reported on Google Code with ID 290

```
Describe your system:
 - Linux distribution: Ubuntu 9.04
 - Machine type: 64-bit
 - Mupen64Plus version: 1.5
 - Plugins used: Rice's Video Plugin 1.5, JttL's SDL Audio 1.5, blight's
SDL input plugin 0.0.10.0, and Hacktarus/Azimer hle rsp plugin

Describe the problem:
When taking multiple screenshots in a row and "On screen display enabled"
is checked, the screenshot capture text will appear in the captured PNGs. 
I'm not familiar with how the graphics work, but my two proposed solutions
are either display the capture text on a different layer of video and
exclude that from the screenshot, or remove any on screen text right before
capturing.
```

Reported by `hansamurai` on 2009-10-22 04:24:58

<hr>
- _Attachment: beetle_adventure_rac-097.png<br>![beetle_adventure_rac-097.png](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-290/comment-0/beetle_adventure_rac-097.png)_

**Comment by richard42** [comment]
```
I looked at this problem and it may not be easy to fix.  I actually wrote the
screenshot code to avoid this problem - it first takes a screenshot after rendering
the display, and then it paints the OSD text.  If you take a lot of screenshots in
a
row, you will see that this bug only shows up in about half of them.  That's because
some or all games actually update at 30fps, though the render function is called once
per field, ie 60 fps, because this was intended to be used on NTSC interlaced TVs.
During the second rendering period the ROM code does not clear the screen or re-draw
anything; it just lets the hardware display the other field from the same (previous)
frame.  There are several different ways that I can think to fix this bug, but they
require significant testing because they may introduce other bugs.
```

Reported by `richard42g` on 2010-01-14 05:00:28
- **Status changed**: `Accepted`
- **Labels added**: Type-Visual, Component-Core

**Comment by richard42** [comment]
Reported by `richard42g` on 2011-05-22 15:56:00
- **Labels added**: Milestone-2.0

**Comment by richard42** [comment]
```
Fixed by updating Video plugin API to 2.1.0
```

Reported by `richard42g` on 2012-03-04 23:17:22
- **Status changed**: `Fixed`
